.github/workflows/api-tests.yml

```diff
@@ -20,7 +20,6 @@ on:
       - "provision/api/**"
       - "readers/api/**"
       - "things/api/**"
-      - "twins/api/**"
       - "users/api/**"
 
```

Comment:
I think this should remain with all services then for services in contrib repo we only check for schema change i.e

```yaml
            notifiers:
              - ".github/workflows/api-tests.yml"
              - "api/openapi/notifiers.yml"
```

This is because the schema files are in core so if there is a change we need to check with schemathesis what is documented is what is implemented. In the setup we can pull contrib and start the services

Reply:
There is no this path anymore, so this change can never be detected.